05-interactive-dst.md

```diff
@@ -23,36 +23,19 @@ is not working as expected.
 The file we [downloaded from the grid](04-files-from-grid.html)
 contains simulated data, with stripping and trigger decisions
 and so on. Here we assumed the file you downloaded is called `00035742_00000002_1.allstreams.dst`.
-To take a look at the contents of the TES we need a simple
-options file:
+To take a look at the contents of the TES we to write a small
```

Comment:
Should be “To take a look at the contents of the TES, we need to write a small…”.